spec/graphql/resolvers/invoice_resolver_spec.rb

```diff
@@ -93,6 +95,28 @@
     expect(group['value']).to eq('usa')
   end
 
+  context 'when invoice has credit notes' do
+    before do
+      create(:credit_note, invoice: invoice)
+    end
+
+    it 'returns an error' do
+      result = execute_graphql(
+      current_user: membership.user,
+      current_organization: organization,
+      query: query,
+      variables: { id: invoice.id },
+    )
+    
+
+    data = result['data']['invoice']
```

Comment:
Layout/IndentationConsistency: Inconsistent indentation detected. (https://rubystyle.guide#spaces-indentation, https://edgeguides.rubyonrails.org/contributing_to_ruby_on_rails.html#follow-the-coding-conventions)